src/main/java/carpet/mixins/LevelAccessor.java

```diff
@@ -0,0 +1,12 @@
+package carpet.mixins;
+
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.redstone.NeighborUpdater;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(Level.class)
+public interface LevelAccessor {
```

Comment:
I suggest you add that to the existing interface class. I know its called WorldInterface - feel free to rename it.

Reply:
omg that's funny, I checked if a levelAccessor already existed. Ya ill rename the current one